proxysql/datadog_checks/proxysql/proxysql.py

```diff
@@ -58,8 +58,12 @@ def __init__(self, name, init_config, instances):
         self.tags.append("proxysql_server:{}".format(self.host))
         self.tags.append("proxysql_port:{}".format(self.port))
 
+        self.version_metadata = is_affirmative(self.instance.get('version_metadata', False))
+
         manager_queries = [STATS_MYSQL_GLOBAL]
         if self.is_metadata_collection_enabled():
+
+            if self.version_metadata:
             # Add the query to collect the ProxySQL version
             manager_queries.append(VERSION_METADATA)
```

Comment:
Since you added an `if`, don't you need to indent more this line like this:
```suggestion
                manager_queries.append(VERSION_METADATA)
```
?